pom.xml

```diff
@@ -7,20 +7,20 @@
     <parent>
         <groupId>com.vaadin</groupId>
         <artifactId>flow-component-base</artifactId>
-        <version>2.0-SNAPSHOT</version>
+        <version>3.1-SNAPSHOT</version>
     </parent>
 
     <properties>
-        <timepicker.version>2.0-SNAPSHOT</timepicker.version>
-        <datepicker.version>2.0-SNAPSHOT</datepicker.version>
+        <timepicker.version>3.1-SNAPSHOT</timepicker.version>
+        <datepicker.version>3.1-SNAPSHOT</datepicker.version>
     </properties>
 
     <artifactId>vaadin-date-time-picker-flow-parent</artifactId>
-    <version>1.0-SNAPSHOT</version>
+    <version>2.0-SNAPSHOT</version>
     <packaging>pom</packaging>
     <name>Vaadin DateTimePicker Flow Parent</name>
 
-   <repositories>
+    <repositories>
```

Comment:
Extra space?

Reply:
No, auto-formatting fixed it from 3 to 4 spaces.